juntagrico/dao/memberdao.py

```diff
@@ -17,6 +17,10 @@ def all_members():
     def members_by_email(email):
         return Member.objects.filter(email__iexact=email)
 
+    @staticmethod
+    def member_by_email(email):
```

Comment:
since the email is unique can't we replace the function above with this one in all places?